python/MooseDocs/base/executioners.py

```diff
@@ -434,21 +405,27 @@ class ParallelBarrier(Executioner):
     def execute(self, nodes, num_threads=1):
         """Perform the translation with multiprocessing."""
 
-        barrier = multiprocessing.Barrier(num_threads)
-        manager = multiprocessing.Manager()
+        # The method for spawning processes changed to "spawn" for macOS in python 3.9, currently
+        # MooseDocs does npt work with this combination. This will be remedied in moosetools
+        # migration.
+        if (platform.python_version() >= '3.9') and (platform.system() == 'Darwin'):
+            ctx = multiprocessing.get_context('fork')
```

Comment:
why is this Darwin specific? googling around I see that people seem to choose different methods (`spawn` over `fork` for macOS) but shouldn't the use of get_context method should only depend on the python version?

Reply:
I'm confused. If the OS is not Darwin, it presumably is Linux, and there `fork` is the default. On mac `spawn` is the default method. So with >=3.9 on macs you're choosing fork over spawn?